homeassistant/components/config/entity_registry.py

```diff
@@ -149,6 +150,7 @@ def _entry_dict(entry):
     return {
         "config_entry_id": entry.config_entry_id,
         "device_id": entry.device_id,
+        "area_id": entry.area_id,
```

Comment:
Should this be moved to the extended dict if there is of no interest of showing it on the entity page?

Reply:
No, we will need the area id in the normal response, will this also include the area id of the device if the entity doesn't have one? If so, we need that info in the extended...

Reply:
Right now the device area id is not included